ReactAndroid/src/main/java/com/facebook/react/views/scroll/ReactHorizontalScrollViewManager.java

```diff
@@ -171,8 +171,12 @@ public void flashScrollIndicators(ReactHorizontalScrollView scrollView) {
   @Override
   public void scrollTo(
       ReactHorizontalScrollView scrollView, ReactScrollViewCommandHelper.ScrollToCommandData data) {
-    if (data.mAnimated) {
-      scrollView.smoothScrollTo(data.mDestX, data.mDestY);
+    if (data.mAnimated && data.mDuration != 0) {
+      if (data.mDuration > 0) {
```

Comment:
Can you add a comment here about -1 being the default platform duration since it's not super obvious from the code. Same thing for the other places this code is copied.